metrics/http.go

```diff
@@ -0,0 +1,15 @@
+package metrics
+
+import (
+	"encoding/json"
+	"net/http"
+)
+
+func HTTPHandler(w http.ResponseWriter, r *http.Request) {
```

Comment:
__[golint]__
* [golint] exported function HTTPHandler should have comment or be unexported

<!-- Comment by [SideCI](https://sideci.com) -->